src/Plugin/Validators/EmptyCell.php

```diff
@@ -78,12 +104,136 @@ public function validateRow($row_values) {
     }
     else {
       $validator_status = [
-        'case' => 'No empty values found in required column(s)',
+        'case' => self::$mapping['case-valid']['dev-case'],
         'valid' => TRUE,
         'failedItems' => [],
       ];
     }
+
     return $validator_status;
   }
 
+  /**
+   * Processes failed validation from EmptyCell into a render array.
+   *
+   * @param array $failures
+   *   An associative array that stores the validation failures by the
+   *   EmptyCell validator. It is keyed by the line number of the input
+   *   file where validation failed, and the value is an associative array
+   *   returned by the validator. Here is the overall structure of $failures:
+   *   - [LINE NUMBER]:
+   *     - 'case': a developer-focused string describing the case checked.
+   *     - 'valid': FALSE to indicate that validation failed.
+   *     - 'failedItems': an array of items that failed:
+   *       - 'empty_indices': A list of column indices in the line which were
+   *         checked and found to be empty.
```

Comment:
For consistency with `GermplasmNameExists` and to avoid repetition, I suggest:
```suggestion
   *     - 'failedItems': an array of items that failed, where the key => value
   *       pairs map to the index => cell value(s) that failed validation.
   *       @see EmptyCell::validateRow()
```

Reply:
The @see comment, should it not be just @see validateRow()? This notation EmptyCell::validateRow() tells me validateRow() is a static method which is not.

Reply:
Just referring to validateRow() sounds good to me! 👍